pageserver/src/tenant/storage_layer/inmemory_layer.rs

```diff
@@ -639,21 +634,18 @@ impl InMemoryLayer {
         // NOTE: this sort can take up significant time if the layer has millions of
         //       keys. To speed up all the comparisons we convert the key to i128 and
         //       keep the value as a reference.
```

Comment:
Can you remove the sort comment as well?

Reply:
fixed in https://github.com/neondatabase/neon/pull/7467/commits/39079a2ab82c38feda1ca23a7aaaeaf99945e3c7